data/txts/tips/amazons.lua

```diff
@@ -29,15 +29,15 @@ tips = {
       seconds = 4
    },
    {
-      text = _"Wilderness keepers ensure fresh game for hunting and fishing. Animals are only released near trees uninhabited by sentries.",
+      text = _"Wilderness keepers ensure fresh game for hunting and fish for fishing. Terrestrial animals are only released near trees uninhabited by sentries.",
       seconds = 4
```

Comment:
suggestion for the last sentence (as we already differentiated game and fish): Instead of "terrestrial animals are" use "Game is only released ..."